perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -213,7 +213,7 @@ sub report_form_ajax : Path('ajax') : Args(0) {
             category        => $category,
             extra_name_info => $extra_name_info,
             titles_list     => $extra_titles_list,
-            categories      => $c->stash->{category_options},
+            categories      => [ map { $_->{name} } @{$c->stash->{category_options}} ], # XXX NEEDED??
```

Comment:
This XXX, is this line needed/used by anything? Was added for mobile apps back in 8e8f83e22823e3cde2403589b5344911ee2a2633 but I can't see anything using it.